### ISSUE TITLE
Images Not Showing at Planet PowerShell

### ISSUE THREAD

**Issue by DarwinJS** [planetpowershell/planetpowershell#107]
I looked around the site and could not figure out how to get this working.

The images in my RSS feed seem to show in most any reader / syndicator - however - at Planet PowerShell they do not.

I am attaching an Innoreader screenshot that shows my direct post with an image and the planet powershell syndicated one without the image.  (The one that starts with "Rugged Tooling")

What RSS image element do I need to add to my feed?

![planetpowershellimages](https://user-images.githubusercontent.com/11597190/39244655-4a613178-485f-11e8-95a7-b795bf1d947d.jpg)

**Comment by kjacobsen** [comment]
Hi,

I have taken a quick look and I can't fully be certain what is going on.

I suspect it has something to do with your RSS feed containing just a summary of the article, but I also can't work out why it appears for other authors who have a similar RSS feed.

**Comment by kjacobsen** [comment]
Hey @DarwinJS is this still an issue that you have seen?

**Comment by DarwinJS** [comment]
Yes it is.  If I view my feed directly in inoreader I see the images - the same articles through the planetpowershell feed do not show the images in inoreader.

I know that my blog's image tagging may not be the most modern - but it is standard.  And if you look closely, there are many other blogs whose banner / title graphic is not showing when proxied through planetpowershell.

Thanks.

**Comment by kjacobsen** [comment]
Thanks! I will keep an eye on it a bit more.
